### DISCUSSION TITLE
有在看什麼2024年4月新番

### DISCUSSION THREAD

**Discussion by 110101501** [1101-GS4719-C/Python-Community-GS4719-C#1963] [General]
如題，近期有些動畫新作或是續作正在開播或是即將開播。
想問大家在這季正在看什麼或者是期待要看什麼呢？

我分享自己這季正在看的作品
1、WIND BREAKER—防風少年—
![231025-wind-01](https://github.com/NCU-GS4719-Python/Python-Community/assets/122419157/743c5c61-458c-498b-9c8f-1f7c7c745368)
漫畫畫風本身就很吸引我，還沒動畫化前就很期待的一部作品。
講述轉入風鈴高中的不良少年櫻遙，想要成為學校第一，加入為了守護城鎮而打架的防風鈴。

2、關於我轉生變成史萊姆這檔事 第3季
![96566c08fe41abaa67f5ed493b1oatb5](https://github.com/NCU-GS4719-Python/Python-Community/assets/122419157/2361b625-6920-47ed-ad04-a29bf1c4e6f3)
萌王利姆路回歸！
以前看這部作品就很喜歡這部平淡的日常風格，而且主角是善良的史萊姆超級可愛。作品中也會有一些有趣的吐槽、對話，觀看起來不會有壓力。

3、約會大作戰Ⅴ
![170798980721](https://github.com/NCU-GS4719-Python/Python-Community/assets/122419157/0505e234-e420-4e2d-bd79-b032033bc333)
進入動漫圈的啟蒙作，根本沒想到動畫會繼續出。怪不得網路上有一句「約戰不倒，陪你到老」。此外，能出到第五季真的很厲害了⋯⋯

4、無職轉生Ⅱ～到了異世界就拿出真本事～ 後半季
![Z7S6oma](https://github.com/NCU-GS4719-Python/Python-Community/assets/122419157/fae8fc27-a9f3-49c0-985c-454d12241261)
魯迪烏斯進入魔法大學後，達成了目標並找回了身為男人的自信。
起初雖然會有點看不慣人物設定，但這部有些事件的反思還不錯，令人印象深刻。

5、我的英雄學院 第七季
![887af66ab981e9d6c7673a5d731o81z5](https://github.com/NCU-GS4719-Python/Python-Community/assets/122419157/2d3d8332-92de-45c7-b5df-74eac76c78c9)
以前就有追這部漫畫，希望動畫也可以出到完結。
內容就不多贅述，但之所以會特別提這部作品的原因是因為這部的片頭曲是由TK from 凛として時雨擔當，是我非常喜歡的一位歌手！

**Comment by willychiuchiu** [comment]
雖然這季新番我都還沒看，不過等之後有空一定要找時間追，謝謝同學分享！我自己目前是在看Jojo第三部，雖然已經很老很老了，但真的很經典啊，歐拉歐拉歐拉！推薦給還沒看過的同學

**Reply by 110101501** [reply]
jojo太經典了…我也超喜歡第三部的埃及團。

**Comment by 20050201Shawn** [comment]
我已經很久沒看新番了，但個人蠻推獵人的，蠻喜歡作者筆下刻畫的每個角色，雖然蟻王篇前段有點躁，但後面很好哭(個人認為)。網路上的也有解說影片，有幾部解析的蠻深刻細膩的，都不知道原來還可以這樣解讀劇情。

**Reply by 110101501** [reply]
獵人也很經典！這部還在我的待看清單中，若有時間也想去補番。

**Comment by EPHMR0524** [comment]
真的好久沒去追新番了，最近沒看到感興趣的作品，上一次追新番應該是去年的星靈感應。
要說今年最期待的作品應該是ATRI吧，雖然說還沒玩完原作遊戲，但還是很期待這部作品的動畫，希望是能趕在動畫出來前能先把遊戲劇情體驗一遍(雖然沒多少時間就是了)

**Reply by 110101501** [reply]
若是沒有時間體驗遊戲劇情的話，也是可以先看完動畫再體驗劇情，說不定會有不同的感受XD。

**Comment by yyujimmy** [comment]
這些作品都是非常受歡迎的，每個人喜歡的類型和風格可能都有所不同。我個人也有一些正在追的動畫，想和你分享一下：

JOJO的奇妙冒險 第6部：石海悍將
這是JOJO系列的續作，故事情節繼續承接上一部。JOJO系列以其奇特的角色設計、獨特的戰鬥風格和濃厚的奇幻色彩而著稱，每一季的主題和故事情節都帶給觀眾新奇的體驗。
刀劍神域 Progressive: Aria of a Starless Night
這是《刀劍神域》系列的新作，以《刀劍神域》系列中的第一季故事為基礎，重新演繹了主角桐人和愛麗絲的冒險。對於喜歡《刀劍神域》系列的粉絲來說，這是一部不容錯過的作品。
蜘蛛精的重生 第2季
這部作品講述了一位少女轉生到了一個充滿怪物和魔法的異世界，並以蜘蛛的身份重新開始生活的故事。劇情跌宕起伏，充滿了懸疑和驚喜，吸引了眾多觀眾的關注。
神之塔 - 王者之盟
這部作品改編自韓國網路漫畫，講述了一群被選中者被困在一座神秘的塔中，為了追求神之力量而進行殘酷的考驗和戰鬥的故事。劇情扣人心弦，充滿了懸念和想像力。
以上是我正在關注的一些動畫，希望你也能找到一些喜歡的作品，一同享受動畫的魅力！
